interop-cats/shared/src/main/scala/zio/stream/interop/fsz2.scala

```diff
@@ -0,0 +1,5 @@
+package zio.stream.interop
+
+object fsz2 extends FS2Platform
```

Comment:
Btw shouldn't it be fs2z? 😄

Reply:
yeah, right, I forgot it is scala**Z** :D